### PR TITLE
avoid unused imports

### PR DESCRIPTION
**Pull request by nineninesevenfour**
fixes https://github.com/eclipse/xtext-xtend/issues/1029

Signed-off-by: Harald Fassler <harald.fassler+9974@gmail.com>

### CODE REVIEW

**Review by szarnekow — APPROVED** [review]
Nice work @nineninesevenfour. Thank you!

File: org.eclipse.xtext.xbase.tests/src/org/eclipse/xtext/xbase/tests/compiler/ImportManagerTest.java [hunk 1]
```diff
@@ -48,9 +48,16 @@ public void setUp() throws Exception {
 		assertTrue(importManager.getImports().isEmpty());
 	}
 
-	@Test public void testOtherString() {
+	@Test public void testOtherStringSamePackage() {
```
[the remark this answers is not in the document]
Comment (nineninesevenfour) [reply org.eclipse.xtext.xbase.tests/src/org/eclipse/xtext/xbase/tests/compiler/ImportManagerTest.java:51]: Should I add the year ("2011, 2020") or replace 2011 by 2020 entirely?
Also in xtext-xtend I found other files, where there are already two years in the header. When there is already written "2014, 2016" for example, should I make "2014, 2016, 2020", "2014-2020" or just "2020"?
Thank you for your help!

[the remark this answers is not in the document]
Comment (cdietrich) [reply org.eclipse.xtext.xbase.tests/src/org/eclipse/xtext/xbase/tests/compiler/ImportManagerTest.java:51]: It is 2011, 2020

[the remark this answers is not in the document]
Comment (cdietrich) [reply org.eclipse.xtext.xbase.tests/src/org/eclipse/xtext/xbase/tests/compiler/ImportManagerTest.java:51]: Or then 2014, 2020

[the remark this answers is not in the document]
Comment (nineninesevenfour) [reply org.eclipse.xtext.xbase.tests/src/org/eclipse/xtext/xbase/tests/compiler/ImportManagerTest.java:51]: OK, thx!

File: org.eclipse.xtext.xbase.testdata/src/bar/String.java [hunk 2]
```diff
@@ -0,0 +1,16 @@
+/*******************************************************************************
+ * Copyright (c) 2011 itemis AG (http://www.itemis.eu) and others.
```
[the remark this answers is not in the document]
Comment (nineninesevenfour) [reply org.eclipse.xtext.xbase.testdata/src/bar/String.java]: done.

File: org.eclipse.xtext.xbase.tests/src/org/eclipse/xtext/xbase/tests/compiler/ImportManagerTest.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (nineninesevenfour) [reply org.eclipse.xtext.xbase.tests/src/org/eclipse/xtext/xbase/tests/compiler/ImportManagerTest.java:51]: done.

**Comment by cdietrich** [comment]
thx @nineninesevenfour

**Comment by nineninesevenfour** [comment]
@cdietrich You are very welcome! Should I also have a look, if `@SuppressWarnings("all")` can now be removed?

**Comment by cdietrich** [comment]
i fear there are still other problems that require the suppresswarnings("all")
but you can have a look for sure

**Comment by cdietrich** [comment]
@nineninesevenfour @szarnekow we now have test fails in Xtend Maven plugin tests with devices with Same name in multiple packages.

these can also be seen in eclipse. will share a repoducer repo

**Pull request merged**